pkg/systemd/services/timer-dialog.jsx

```diff
@@ -271,14 +314,33 @@ const CreateTimerDialogBody = ({ owner }) => {
                                 validated = "error";
                                 helperTextInvalid = _("Minute needs to be a number between 0-59");
                             }
+                            const sec = repeatPatterns[idx].second;
+                            const validationFailedSecond = !(/^[0-9]+$/.test(sec) && sec <= 59 && sec >= 0);
+
+                            if (submitted && repeat == 'minutely' && validationFailedSecond) {
+                                validated = "error";
+                                helperTextInvalid = _("Second needs to be a number between 0-59");
+                            }
 
                             return (
                                 <FormGroup label={label} key={item.key}
                                            validated={validated}
                                            helperTextInvalid={helperTextInvalid}>
                                     <Flex className="specific-repeat-group" data-index={idx}>
+                                        {repeat == "minutely" && <>
+                                            <TextInput className='delay-number'
+                                                       id={repeat}
+                                                       value={repeatPatterns[idx].second}
+                                                       onChange={second => {
+                                                           const arr = [...repeatPatterns];
+                                                           arr[idx].second = second;
+                                                           setRepeatPatterns(arr);
+                                                       }}
+                                                       validated={submitted && validationFailedSecond ? "error" : "default"} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17592-20220809-115752-7fb44acf-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)